client.go

```diff
@@ -1015,6 +1017,22 @@ func (client *gocloak) GetGroups(token string, realm string, params GetGroupsPar
 	return result, nil
 }
 
+// GetGroupsCount gets the groups count in the realm
+func (client *gocloak) GetGroupsCount(token string, realm string) (int, error) {
+	const errMessage = "could not get groups count"
+
+	var result int
```

Comment:
so the result is json object:
```json
{
       "count": 0
}
```
Please add such structure

Reply:
I mean the keycloak returns a json, not just an integer. So you need to create a structure, like
```go
type GroupsCount struct {
    Count int `json:"count"`
}
```

the use it here:
```go
var result GroupsCount
...
return result.Count
```